tests/integration-tests/pom.xml

```diff
@@ -126,12 +126,6 @@
       </dependency>
       <dependency>
          <groupId>org.apache.activemq</groupId>
-         <artifactId>artemis-proton-plug</artifactId>
-         <version>${project.version}</version>
-         <type>test-jar</type>
-      </dependency>
```

Comment:
FYI, the other `<dependency>` is not an exact duplicate, it defines a `<scope>test</scope>`.